packages/e2e/cypress/e2e/api/organizationPermissions.cy.ts

```diff
@@ -251,14 +251,16 @@ describe('Lightdash API tests for viewer org user', () => {
         });
     });
 
-    it('Should get a forbidden (403) from POST project', () => {
+    it('Should get a forbidden error (403) from POST project', () => {
```

Comment:
viewers shoudl also get a 403 error for previews, right ? should we "duplicate" this test then ?

Reply:
updated